packages/@cdktf/hcl2cdk/test/__snapshots__/iteration.test.ts.snap

```diff
@@ -34,7 +35,7 @@ const awsRoute53RecordExample = new aws.route53Record.Route53Record(
     records: [\\"\${each.value.record}\\"],
     ttl: 60,
     type: \\"\${each.value.type}\\",
-    zoneId: dataAwsRoute53ZoneExample.zoneId,
+    zoneId: cdktf.Token.asString(dataAwsRoute53ZoneExample.zoneId),
```

Comment:
Why is this wrapped? `zoneId` [already returns a string token](https://github.com/cdktf/cdktf-provider-aws/blob/eba8748e62f84d95bbc639703e6aabd6277cf7d3/src/data-aws-route53-zone/index.ts#L226) 👀

Reply:
I looked into it and it's because of the order the parser returns on the resources, the resource does not exist yet at the point in time when the valueToTs function is called apparently. I think this is an more invasive change than I'd like to do in this PR so I'd say we scope this out into a follow up and discuss the prioritisation then. @mutahhir is that ok with you?

Reply:
Sounds good to me. Let's create an issue for this.